airbyte-config/config-models/src/main/java/io/airbyte/config/exception/ConnectionFileServiceErrorException.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.config.exception;
+
+public class ConnectionFileServiceErrorException extends RuntimeException {
```

Comment:
i did not find usages of this exception. Do we need this class?

Reply:
valid remark. removed it